tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_tests.go

```diff
@@ -0,0 +1,48 @@
+package dataapigeneratorjson
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/hashicorp/pandora/tools/sdk/resourcemanager"
+)
+
+type TerraformResourceTestConfig struct {
+	Basic          string  `json:"Basic"`
+	RequiresImport string  `json:"RequiresImport"`
+	CompleteConfig *string `json:"CompleteConfig,omitempty"`
+	TemplateConfig *string `json:"TemplateConfig,omitempty"`
```

Comment:
it's possible for us to have other tests here too, so we need to support those:

```suggestion
	TemplateConfig *string `json:"TemplateConfig,omitempty"`
	OtherTests *map[string]string `json:"OtherTests,omitempty"`
```